cmd/sealer/cmd/image/build.go

```diff
@@ -78,6 +78,9 @@ func NewBuildCmd() *cobra.Command {
 		},
 	}
 	buildCmd.Flags().StringVarP(&buildFlags.Kubefile, "file", "f", "Kubefile", "Kubefile filepath")
+	//todo we can support imageList Flag to download extra container image rather than copy it to rootfs
+	buildCmd.Flags().StringVar(&buildFlags.ImageList, "image-list", "filepath", "`pathname` of imageList filepath, if set, sealer will read its content and download extra container")
```

Comment:
what's the difference for `image-list` and `image-list-with-auth`?

And how about combine them?

Reply:
1. image-list: extra container images list which is no auth required need to be saved to clusterimage. 
2. image-list-with-auth: extra container images list which is auth required need to be saved to clusterimage,  see example: http://sealer.cool/docs/getting-started/build-cloudimage.html#more-build-examples

Reply:
what's the purpose for store the image list at present?